src/Common/EventListener/ResponseSecurer.php

```diff
@@ -14,16 +14,38 @@ class ResponseSecurer
      */
     public function onKernelResponse(FilterResponseEvent $event): void
     {
+        $cacheControl = 'max-age=0, must-revalidate, private, no-cache';
+        if (defined('APPLICATION') && APPLICATION === 'Backend') {
+            $cacheControl .= ', no-store';
+        }
+
         $headers = [
+            'Cache-Control' => $cacheControl,
             'X-Frame-Options' => 'deny',
+            'Expires' =>  '0',
+            'Pragma' => 'no-cache',
```

Comment:
is this needed, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Pragma
> Use Pragma only for backwards compatibility with HTTP/1.0 clients.

Reply:
I added it here because it came back in an audit done on a fork project

Reply:
ok.